test/jest/kitFilterHelper.spec.ts

```diff
@@ -168,6 +168,105 @@ describe('FilterHashingUtilities', () => {
         });
     });
 
+    describe('#hashEventMessage', () => {
+        it('should hash event message for SessionStart', () => {
+            const eventName = 'Session Start';
+            const eventType = EventType.Other;
+            const messageType = MessageType.SessionStart;
+            const resultHash = KitFilterHelper.hashEventMessage(eventName, eventType, messageType);
+            expect(resultHash).toBe(1006044049);
```

Comment:
session start and session end are not the best examples for this, because their event name is technically not correct. 

They are numbers.  Here's session start, as an example.  

<img width="294" height="138" alt="Image" src="https://github.com/user-attachments/assets/f4ab15fb-9a6e-4fc9-b8d0-e0464d2a0daf" />

I don't think the examples _have_ to be session start and session end, and you were doing it just to change the message type, is that right?